extended/src/test/java/io/kubernetes/client/extended/workqueue/DefaultDelayingQueueTest.java

```diff
@@ -103,7 +103,6 @@ private boolean waitForAdded(DefaultDelayingQueue queue, int size) {
   }
 
   private boolean waitForWaitingQueueToFill(DefaultDelayingQueue queue) {
-    return Wait.poll(
```

Comment:
I don't think these are the same test. (and it seems unrelated) can you revert this change?

Reply:
changed it back...